docs/ug/configFiles.md

```diff
@@ -59,6 +59,17 @@ When using [standalone config](#config-json-standalone-config-file) (if it is no
 
 Optionally, you can use an `author-config.csv` (which should be in the same directory as the `repo-config.csv` file) to provide more details about the authors to analyze ([example](author-config.csv)). It should contain the following columns:
 
+<<<<<<< HEAD
```

Comment:
The merge conflict is still here

Reply:
Well that's embarrassing. I could have sworn I added the changes before committing. 

> After looking through the backend code again, I noticed that some files in model still refer to GitHub (such as Author#STANDARD_GITHUB_EMAIL_DOMAIN, RepoLocation#GIT_REPOSITORY_LOCATION_PATTERN and StandaloneAuthor#githubId)

Regarding this, I do not think it is wise to do the renaming of variables here because there are in fact a lot of specific references to GitHub both in variable/method names, string constants and DG. It would add a sizeable amount of lines changed to this PR without really affecting any functionality. As such, I edited only the parts I felt were most immediately relevant parts in the UG. 

@yhtMinceraft1010X requesting follow-up review